Ensure LLVM and CLang are found, changed pybind11 version to 1.11
Author J. Sassmannshausen (Imperial College London)
diff --git a/hoomd-4.9.1.orig/CMake/hoomd/HOOMDPythonSetup.cmake b/hoomd-4.9.1/CMake/hoomd/HOOMDPythonSetup.cmake
index 780344e..7d9e544 100644
--- a/hoomd-4.9.1.orig/CMake/hoomd/HOOMDPythonSetup.cmake
+++ b/hoomd-4.9.1/CMake/hoomd/HOOMDPythonSetup.cmake
@@ -12,7 +12,7 @@ if (Python_FOUND)
     endif()
 endif()
 
-find_package(pybind11 2.12 CONFIG REQUIRED)
+find_package(pybind11 2.11 CONFIG REQUIRED)
 
 if (pybind11_FOUND)
     find_package_message(pybind11 "Found pybind11: ${pybind11_DIR} ${pybind11_INCLUDE_DIR} (version ${pybind11_VERSION})" "[${pybind11_DIR}][${pybind11_INCLUDE_DIR}]")
diff --git a/hoomd-4.9.1.orig/hoomd/hpmc/CMakeLists.txt b/hoomd-4.9.1/hoomd/hpmc/CMakeLists.txt
index 0174fb0..eca7595 100644
--- a/hoomd-4.9.1.orig/hoomd/hpmc/CMakeLists.txt
+++ b/hoomd-4.9.1/hoomd/hpmc/CMakeLists.txt
@@ -226,13 +226,8 @@ if (ENABLE_LLVM)
     find_package(LLVM REQUIRED CONFIG)
 
     if (LLVM_FOUND)
-        find_library(llvm_library LLVM
-                     HINTS ${LLVM_LIBRARY_DIRS}
-                     NO_DEFAULT_PATH)
-
-        find_library(clang_library clang-cpp
-                     HINTS ${LLVM_LIBRARY_DIRS}
-                     NO_DEFAULT_PATH)
+        find_library(llvm_library LLVM)
+        find_library(clang_library clang-cpp)
     endif()
     find_package_message(LLVM "Found LLVM: ${LLVM_DIR} ${llvm_library} ${clang_library} ${LLVM_INCLUDE_DIRS} ${LLVM_DEFINITIONS} (version ${LLVM_PACKAGE_VERSION})" "[${LLVM_DIR},${llvm_library},${clang_library}]")
 
